vertical-pod-autoscaler/README.md

```diff
@@ -50,104 +50,10 @@ existing recommendations will be removed. The objects will remain present though
 and a ConfigUnsupported condition will be set on them.
 
 To receive recommendations please migrate to `autoscaling.k8s.io/v1beta2` API,
-as described in the next section.
-Please also have a look there if you want to understand the rationale behind
-the API change.
+as described in the [Migration Doc](https://github.com/kubernetes/autoscaler/blob/vpa-release-0.3/vertical-pod-autoscaler/MIGRATE.md)
 
-### Notice on switching to v1beta2 version (0.3.X to >=0.4.0)
+This doc is for installing latest VPA. For instructions on migration from older versions see Migration Doc [Migration Doc](https://github.com/kubernetes/autoscaler/blob/vpa-release-0.3/vertical-pod-autoscaler/MIGRATE.md)
```

Comment:
nit: You can remove the first Migration Doc (this will only leave the link)

Reply:
Another thing, let's lint to the master branch, since the vpa-release-0.3 does not have MIGRATE.md